src/test/storage/single_segment_index_test.cpp

```diff
@@ -40,34 +128,436 @@ class SingleSegmentIndexTest : public BaseTest {
     return result;
   }
 
-  std::shared_ptr<BaseIndex> index_int = nullptr;
-  std::shared_ptr<BaseSegment> dict_segment_int = nullptr;
-  std::shared_ptr<BaseIndex> index_str = nullptr;
-  std::shared_ptr<BaseSegment> dict_segment_str = nullptr;
+  // segments
+  std::shared_ptr<BaseSegment> dict_segment_int_no_nulls = nullptr;
```

Comment:
Declare them in a list to save some space?

Reply:
On the other hand, it would reduce the comprehensibility if the descriptive names are missing in the test cases.